assets/locales/en.json

```
@@ -149,5 +150,41 @@
   "cantAddIpfsToPath": {
     "title": "IPFS on PATH",
     "message": "Could not add IPFS to the PATH."
+  },
+  "launchAtLoginNotSupported": {
+    "title": "Launch at login",
+    "message": "Launch at login is not supported on your platform."
+  },
+  "launchAtLoginFailed": {
+    "title": "Launch at login",
+    "message": "Launch at login could not be enabled on your machine."
+  },
+  "ipfsOnPathInstall": {
+    "title": "Install IPFS on PATH",
+    "message": "By enabling this option, IPFS will be available on your command line as \"ipfs\". This action is reversible.",
+    "action": "Install"
+  },
+  "ipfsOnPathUninstall": {
+    "title": "Uninstall IPFS on PATH",
+    "message": "By disabling this option, IPFS will no longer be available on your command line as \"ipfs\".",
+    "action": "Uninstall"
+  },
+  "enableGlobalTakeScreenshotShortcut": {
+    "title": "Take screenshot shortcut",
```

```suggestion
    "title": "Enable screenshot shortcut",
```
(assuming we are using "enable" language)